src/Proxy/Health/Node.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace OpenRuntimes\Proxy\Health;
+
+class Node
+{
+    private string $hostname;
+
+    private bool $online;
+
+    /**
+     * @var array<string,mixed>
+     */
+    private array $state;
+
+    /**
+     * @param array<string,mixed> $state
+     */
+    public function __construct(string $hostname, bool $online = false, array $state = [])
+    {
+        $this->hostname = $hostname;
+        $this->online = $online;
+        $this->state = $state;
+    }
+
+    public function getHostname(): string
```

Comment:
we're missing doc blocks for these functions

Reply:
Is this a concern? I tried to prevent doing doc blocks unless necessary. With PHP 8 everything is properly typed by types, except arrays. For arrays I included doc blocks.

This made PHP Stan check pass on max level.

Reply:
For consistency, Ill add proper doc blocks everywhere.